lib/types.ts

```diff
@@ -284,7 +304,7 @@ export interface SpaceAPI {
   ) => Promise<CollectionResponse<T>>
 
   getAsset: <T = Object>(id: string) => Promise<T>
-  getAssets: <T = Object, InputArgs = SearchQuery>(
+  getAssets: <T = Object, InputArgs = SearchQuery>( // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
We should change this to match getEntries on line 292 like so 
```suggestion
  getAssets: <T = Object, InputArgs = SearchQuery>(query?: InputArgs) => Promise<CollectionResponse<T>>
```

or potentially just remove `InputArgs` entirely, and 
```suggestion
  getAssets: <T = Object>(query?: SearchQuery) => Promise<CollectionResponse<T>>
```

Reply:
Yes, I agree that we should remove the `eslint-disable-line`. I added it because eslint showed a warning for `InputArgs` being unused which doesn't make sense in this case as it's a type definition.

Reply:
<img width="1191" alt="Screenshot 2020-11-23 at 15 34 52" src="https://user-images.githubusercontent.com/4947671/99974587-702e5000-2da1-11eb-8496-268f8b8b1728.png">